man/systemd.exec.xml

```diff
@@ -859,14 +859,16 @@
 
       <varlistentry>
         <term><varname>PAMName=</varname></term>
-        <listitem><para>Sets the PAM service name to set up a session
-        as. If set, the executed process will be registered as a PAM
-        session under the specified service name. This is only useful
-        in conjunction with the <varname>User=</varname> setting. If
-        not set, no PAM session will be opened for the executed
-        processes. See
-        <citerefentry project='man-pages'><refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum></citerefentry>
-        for details.</para></listitem>
+        <listitem><para>Sets the PAM service name to set up a session as. If set, the executed process will be
+        registered as a PAM session under the specified service name. This is primarily useful in conjunction with the
```

Comment:
This changed "only useful with" with "primarily useful with". But, [this option is ignored if no username is specified](https://github.com/systemd/systemd/blob/357cd5563a40a65306901ea2c7cbf121244b76e9/src/core/execute.c#L2706):

```c
if (context->pam_name && username) {
    r = setup_pam(context->pam_name, username, uid, gid, context->tty_path, &accum_env, fds, n_fds);
```